tests/functional/spec/timeout.spec.js

```diff
@@ -1,12 +1,13 @@
 const helpers = require('../helpers');
-const testTimeoutJSON = require('../../../data/en/test_timeout.json');
+const TimeoutBlockPage = require('../generated_pages/timeout/timeout-block.page.js');
 
 describe('Timeout', function() {
-  it('Given I am completing an electronic questionnaire, when I have been inactive for X minutes and attempt to submit data, then I will be redirected to a page confirming my session has timed out', function() {
+  it('Given I am completing an electronic questionnaire, when I have been inactive for the timeout period and attempt to submit data, then I will be redirected to a page confirming my session has timed out', function() {
     return helpers.openQuestionnaire('test_timeout.json').then(() => {
-      return browser.pause(testTimeoutJSON.session_timeout_in_seconds * 1000).then(() => {
-        return helpers.pressSubmit(1).getSource().should.eventually.contain('Your session has expired');
-      });
+      return browser
+          .pause(3000)
```

Comment:
This is right on the limit now that you've increased the schema timeout so it might be worth increasing this slightly but happy to approve and you can just do that with the squash.